.github/workflows/release_production.yml

```
@@ -65,35 +65,29 @@ jobs:
 
   send_slack_failure:
     name: Send Release Slack notification failure
-    runs-on: ubuntu-latest
+    runs-on: Runner_8cores_Deriv-app
+    environment: Production
     if: failure()
     needs: [build_test_and_publish]
     steps:
-      - name: Create Slack Message
-        id: create_slack_message
-        run: |
-          echo "MESSAGE=${{ env.RELEASE_TYPE }} Release failed for app.deriv.com with version ${{ needs.build_test_and_publish.outputs.RELEASE_VERSION }}" >> $GITHUB_OUTPUT
       - name: Send Slack Notification
-        uses: "./.github/actions/send_slack_notifications"
+        uses: "deriv-com/shared-actions/.github/actions/send_slack_notification@master"
         with:
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
-          MESSAGE: ${{ steps.create_slack_message.outputs.MESSAGE }}
+          MESSAGE: "${{ env.RELEASE_TYPE }} Release failed for app.deriv.com with version ${{ needs.build_test_and_publish.outputs.RELEASE_VERSION }}"
```

```suggestion
          MESSAGE: "$RELEASE_TYPE Release failed for app.deriv.com with version ${{ needs.build_test_and_publish.outputs.RELEASE_VERSION }}"
```
is it working like this? 🤔